src/ssl.c

```diff
@@ -2383,6 +2386,11 @@ int wolfSSL_Rehandshake(WOLFSSL* ssl)
         }
 #endif
 
+        if (!resume) {
+            XMEMSET(ssl->session.sessionID, 0, ID_LEN);
```

Comment:
Consider `sizeof(ssl->session.sessionID)` instead of `ID_LEN`? Either is fine.

Reply:
Making the change to sizeof.